src/main/java/com/cloudant/client/api/model/FindByIndexOptions.java

```diff
@@ -142,7 +148,13 @@ public Integer getReadQuorum() {
     }
 
     public String getUseIndex() {
-        return useIndex;
+        return useIndex.toString();
     }
 
+    /**
+     * For use in {@link Database#getFindByIndexBody(String, FindByIndexOptions)}
+     */
+    public JsonArray getUseIndexAsJsonArray() {

```

Comment:
I'm reluctant to add a public method to this API class.
Maybe just make the field public?